gherkin-tests/src/main.rs

```diff
@@ -0,0 +1,107 @@
+use std::{cmp::Ordering, sync::Arc};
+
+use cucumber::{given, then, when, WorldInit};
+use many_client::client::ledger::TokenAmount;
+use num_bigint::BigUint;
+use opts::{read_spec_config, CmdOpts};
+use world::{IdentityName, World};
+
+mod cose;
+mod opts;
+mod world;
+
+#[given(expr = "an identity {identity}")]
```

Comment:
Let's call it `identifier`. It's more semantically correct and we might want to identify others.

In the case of `identity`, reading the code, I'd expect to see a textual address (the `m...` string).